src/Symfony/Component/AssetMapper/MappedAsset.php

```diff
@@ -76,19 +84,22 @@ public function __construct(
         foreach ($fileDependencies as $fileDependency) {
             $this->addFileDependency($fileDependency);
         }
+        foreach ($javaScriptImports as $javaScriptImport) {
+            $this->addJavaScriptImport($javaScriptImport);
+        }
```

Comment:
can't we make this shorter and faster?
```suggestion
        $this->fileDependencies = $fileDependencies;
        $this->addJavaScriptImports = $javaScriptImports;
```